.github/workflows/security-update-check.yml

```
@@ -0,0 +1,43 @@
+          tags: blog-payload:security-check
+          
+      - name: Scan for updates on CVE-2025-4802
+        run: |
+          echo "🔍 Checking if CVE-2025-4802 has been patched..."
+          docker run --rm -v /var/run/docker.sock:/var/run/docker.sock \
+            aquasec/trivy:latest image \
+            --severity HIGH,CRITICAL \
+            --format json \
+            'blog-payload:security-check' | \
+            jq '.Results[]?.Vulnerabilities[]? | select(.VulnerabilityID == "CVE-2025-4802")' > jq_output
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)